src/builtin/functions/meta.rs

```diff
@@ -101,30 +99,74 @@ pub(crate) fn global_variable_exists(
     mut args: CallArgs,
     parser: &mut Parser<'_>,
 ) -> SassResult<Value> {
-    args.max_args(1)?;
-    match args.get_err(0, "name")? {
-        Value::String(s, _) => Ok(Value::bool(parser.global_scope.var_exists(s.into()))),
-        v => Err((
-            format!("$name: {} is not a string.", v.inspect(args.span())?),
-            args.span(),
-        )
-            .into()),
-    }
+    args.max_args(2)?;
+
+    let name: Identifier = match args.get_err(0, "name")? {
+        Value::String(s, _) => s.into(),
+        v => {
+            return Err((
+                format!("$name: {} is not a string.", v.inspect(args.span())?),
+                args.span(),
+            )
+                .into())
+        }
```

Comment:
These patterns looks weird and not ergonomic though, I wonder if we can improve it?

```rust
        v => return SassError::new(
            format!("$name: {} is not a string.", v.inspect(args.span())?),
            args.span(),
        ),
```

Or maybe something like that? I don't think the one I suggested is good but the `.into()` here looks weirder.

Reply:
We use that pattern currently to make it easier to change the error type. This was useful, for example, when we boxed all errors.